src/Libraries/CoreNodes/List.cs

```diff
@@ -819,6 +819,56 @@ public static bool AllFalse(IList list)
             return result;
         }
 
+        /// <summary>
+        ///     Determines if any item in the given list is a boolean and has a true value.
+        /// </summary>
+        /// <param name="list">List to be checked on whether any item is true.</param>
+        /// <returns name="bool">Whether any item is true.</returns>
+        /// <search>test,any,true,istrue</search>
```

Comment:
Why use `test` keyword here ?

Reply:
Ah OK I see in history we use `test` some nodes in same cate, but it does not make sense to me.. I will check with team